cmd/juju/commands/bootstrap.go

```diff
@@ -329,9 +331,12 @@ func (c *bootstrapCommand) SetFlags(f *gnuflag.FlagSet) {
 	f.BoolVar(&c.noSwitch, "no-switch", false, "Do not switch to the newly created controller")
 	f.BoolVar(&c.Force, "force", false, "Allow the bypassing of checks such as supported series")
 	f.BoolVar(&c.noHostedModel, "no-default-model", false, "Do not create a default model")
-	f.StringVar(&c.ControllerCharmPath, "controller-charm", "", "Path to a locally built controller charm")
+	f.StringVar(&c.ControllerCharmPath, "controller-charm-path", "", "Path to a locally built controller charm")
+	f.StringVar(&c.ControllerCharmRisk, "controller-charm-risk", "beta", "The controller charm risk if not using a local charm")
 }
 
+var controllerCharmRisks = set.NewStrings("edge", "beta", "candidate", "stable")
```

Comment:
Is there a need to put these constants in a separate package that represents the business for the controller charm?

Reply:
I've changed to use the Risk values from the charms repo